### PR TITLE
add feature precourse

### PR DESCRIPTION
**Pull request by carlosq09**

### CODE REVIEW

File: staff/carlos-luis-querecuto/airlines/index.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        (flights[i].scale)? (modal+=" With scale/s \n",flightsws++) : modal+=" No scale \n"  ;
+    }
+    return modal;
+}
+function admin(name,flights) {
+    var loop = true;
+    var op , todo, sorted;
+    var newflight = [
+        { id: 00, to: '', from: '', cost: 0, scale: false }
+    ]
```
Comment (Markuson) [inline staff/carlos-luis-querecuto/airlines/index.js]: missing ";"

File: staff/carlos-luis-querecuto/airlines/index.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            if(flights.length <= 15){
+                newflight.id=flights.length;
+                newflight.to=prompt("Insert departure");
+                newflight.from=prompt("Insert destination");
+                newflight.cost=prompt("Insert price").toString();
+                newflight.scale=confirm("Have scales?");
+                flights.push(newflight);
+                alert("flight added Successfully! \n"+dataHandler(flights));
+            }else{
+                alert("Max quantity of Flights allowed are 15!")
```
Comment (Markuson) [inline staff/carlos-luis-querecuto/airlines/index.js]: missing ";"

File: staff/carlos-luis-querecuto/airlines/index.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            return buy;
+        }
+        else{
+            alert("Sorry! we cant find a flight with that id :(");
+            find=sorted;
+        }
+    }
+}
+function skylabAirlines() {
+    var name, userBuy;
```
Comment (Markuson) [inline staff/carlos-luis-querecuto/airlines/index.js]:
```
var name,
      userBuy;

```
should be more polite.

File: staff/carlos-luis-querecuto/airlines/index.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        { id: 06, to: 'Madrid', from: 'Barcelona', cost: 90, scale: false },
+        { id: 07, to: 'Tokyo', from: 'Madrid', cost: 1500, scale: true },
+        { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+        { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+        { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false }
+    ];
+    name=prompt("Welcome to SkylabAirlines Service, What's your name?");
+    confirm(flightsDisplay(flights,name));
+    if(prompt("If youre admin, please type 'ADMIN', if not type a key to continue as a user")==='ADMIN'){
+        admin(name,flights)
```
Comment (Markuson) [inline staff/carlos-luis-querecuto/airlines/index.js]: missing ";"

File: staff/carlos-luis-querecuto/airlines/index.js (hunk 3 above)
Comment (carlosq09) [reply staff/carlos-luis-querecuto/airlines/index.js]: those variables are not assigned yet, maybe in only a line confussions could be avoided.

File: staff/carlos-luis-querecuto/airlines/index.js (hunk 1 above)
Comment (carlosq09) [reply staff/carlos-luis-querecuto/airlines/index.js]: fixed

File: staff/carlos-luis-querecuto/airlines/index.js (hunk 2 above)
Comment (carlosq09) [reply staff/carlos-luis-querecuto/airlines/index.js]: fixed

File: staff/carlos-luis-querecuto/airlines/index.js (hunk 4 above)
Comment (carlosq09) [reply staff/carlos-luis-querecuto/airlines/index.js]: fixed

**Pull request merged**